src/script/vco2_main/factions/wh2_dlc15_hef_imrik/campaigns/main_warhammer/missions.lua

```diff
@@ -1,186 +1,160 @@
+            victory_type vco_victory_type_short;
+            key wh_main_short_victory;
+            issuer CLAN_ELDERS;
+            primary_objectives_and_payload
+            {
 
-				objective
-				{
-					type COMPLETE_N_MISSIONS_OF_CATEGORY;
-					total 3;
-					event_category Objective;
-				}
-				objective
-				{
-					type CONTROL_N_PROVINCES_INCLUDING;
-					total 5;
-					province wh2_main_caledor;
-					province wh2_main_dragon_isles;
-					province wh2_main_gnoblar_country;
-					province wh2_main_the_plain_of_bones;
-					province wh2_main_the_wolf_lands;
-				}
-				objective
-				{
-					type DESTROY_FACTION;
-					faction wh2_main_skv_clan_eshin;
+                objective
+                {
+                    type COMPLETE_N_MISSIONS_OF_CATEGORY;
+                    total 3;
+                    event_category Objective;
+                }
+                objective
+                {
+                    type CONTROL_N_PROVINCES_INCLUDING;
+                    total 5;
+                    province wh2_main_caledor;
+                    province wh2_main_dragon_isles;
+                    province wh2_main_gnoblar_country;
+                    province wh2_main_the_plain_of_bones;
+                    province wh2_main_the_wolf_lands;
+                }
+                objective
+                {
+                    type DESTROY_FACTION;
+                    faction wh2_main_skv_clan_eshin;
 
-					faction wh2_main_def_hag_graef;
-					confederation_valid;
-				}
-				objective
-				{
-					type RAZE_OR_SACK_N_DIFFERENT_SETTLEMENTS_INCLUDING;
-					total 1;
-					region wh_main_death_pass_karak_drazh;
-				}
-				objective
-				{
-					type CONSTRUCT_N_BUILDINGS_FROM;
-					total 3;
-					building_level wh2_main_hef_mages_3;
-					building_level wh2_main_hef_worship_3;
-					building_level wh2_main_hef_dragons_2;
-					building_level wh2_main_hef_smith_2;
-					building_level wh2_dlc10_hef_handmaidens;
-					building_level wh2_main_hef_defence_major_3;
-					building_level wh2_main_hef_order_5;
-					building_level wh2_main_hef_port_3;
-					faction wh2_main_hef_caledor;
-				}
-				objective
+                    faction wh2_main_def_hag_graef;
+                    confederation_valid;
+                }
+                objective
+                {
+                    type RAZE_OR_SACK_N_DIFFERENT_SETTLEMENTS_INCLUDING;
+                    total 1;
+                    region wh_main_death_pass_karak_drazh;
+                }
+                objective
                 {
                     type CONSTRUCT_N_BUILDINGS_INCLUDING;
                     faction wh2_dlc15_hef_imrik;
-                    total 1;
+                    total 2;
                     building_level wh2_dlc15_special_graves_of_the_dragons_2_hef;
+                    building_level wh2_main_hef_dragons_2;
+                }
+                objective
+                {
+                    type OWN_N_UNITS;
+                    total 5;
```

Comment:
I'll change the Dragon unit requirements from 5/15 to 4/8. We can discuss it in the PR thread if you want to.